server/app/models/user.rb

```diff
@@ -21,4 +24,17 @@ def name
     self.email
   end
 
+  ##
+  # @param [String] role
+  def in_role?(role)
+    self.roles.where(name: role).exists?
+  end
+
+  def master_admin?
+    self.in_role?('Master admin')
```

Comment:
I have changed this so that we have constants now for those roles: https://github.com/kontena/kontena/blob/feature/user_permissions/server/app/models/user.rb#L9-L10